UPGRADE-7.4.md

```diff
@@ -18,6 +18,11 @@ Console
 
  * Deprecate `Symfony\Component\Console\Application::add()` in favor of `addCommand()`
 
+BrowserKit
+----------
+
+ * Leverage the native HTML5 parser when using PHP 8.4+
```

Comment:
this is a BC break for code interacting with DomCrawler methods returning a raw node as the type has changed. this should be documented as such.